src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -944,6 +944,155 @@ public virtual RedirectToRouteResult RedirectToRoutePermanent(string routeName,
         }
 
         /// <summary>
+        /// Redirects (<see cref="StatusCodes.Status302Found"/>) to the current page.
+        /// </summary>
+        /// <returns>The <see cref="RedirectToPageResult"/>.</returns>
+        [NonAction]
+        public virtual RedirectToPageResult RedirectToPage()
+            => RedirectToPage(pageName: null);
+
+        /// <summary>
+        /// Redirects (<see cref="StatusCodes.Status302Found"/>) to the current page with the specified <paramref name="routeValues"/>.
```

Comment:
> to the current page

If this overload is meant to redirect to the same page but with different `routeValues` we should leave it off of `ControllerBase` right?